scripts/generate-app-icon.py

```
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
+
+import os
+from PIL import Image, ImageDraw, ImageFont
```

Ruff (0.12.2)

`PIL.ImageFont` imported but unused

Remove unused import: `PIL.ImageFont`

(F401)